src/pilot/assign_role.py

```diff
@@ -983,6 +985,9 @@ def get_drives(drac_client):
 
     if physical_disks:
         for pd_id in physical_disks:
+            # Get all NVMe drives
+            if is_nvme_drive(physical_disks[pd_id]):
+                nvme_drives.append(physical_disks[pd_id])
```

Comment:
I think you need a continue above.  Otherwise, line 993 will be executed below.  The raid_status will be None for NVMe drives, so the if statement will be entered, then it will log the skipping message even though it did not skip the drive, then continue.

Reply:
Done.